R/utils.R

```diff
@@ -110,3 +110,31 @@ resolve_duplicates <- function(x) {
            " duplicate(s) found. Please resolve duplicate records."))
   }
 }
+
+#' Compute and display tree-level fire history statistics
+#' @param An fhx instance
+#' @return A data.frame providing information about each tree in the fhx instance
+tree.stats <- function(fhx){
```

Comment:
We're going to want to remove that period to avoid clashes with S3 methods. "tree_stats", "treestats", "stats_tree", or "statstree" are alternatives.